src/pages/Onboard/NameTeam.vue

```diff
@@ -130,18 +136,92 @@ export default {
       await this.createLicense()
       if (!this.updateServerError) this.goToResources()
     },
-    goToResources() {
+    async accept(pt) {
+      try {
+        this.accepting = true
+        const invitationId = pt.id
+        const accepted = await this.acceptMembershipInvitation(invitationId)
+        this.redirectTenant = pt.tenant.slug
+        if (accepted.accept_membership_invitation.id) {
+          this.redirectTenant = pt.tenant.slug
+          this.accepting = false
+        }
+      } catch (e) {
+        this.mutationErrorMessage = e
+          .toString()
+          .split(':')
+          .pop()
+        this.loading = false
+        this.error = true
+      }
```

Comment:
Could we make the pending invitations list update sooner - possibly be refetching the pending invitations query here?  There's a bit of a lag which really shows up if there's more than one invitation. 



![screengif](https://user-images.githubusercontent.com/40272060/97016465-c39b4d00-151a-11eb-843f-8415a9901941.gif)